topics/single-cell/tutorials/scrna-data-formats/slides.html

```diff
@@ -1,585 +1,44 @@
-CellXGene Collection | https://cellxgene.cziscience.com/datasets 
-Single Cell Portal | https://singlecell.broadinstitute.org/single_cell
-EBI Single Cell Expression Atlas | https://www.ebi.ac.uk/gxa/sc/home
-
-???
-
-There are many publicly available sources for reusable single-cell data. The table shows some common sources for acquiring this data.
-
-This table is not extensive and there are many other resources available!
-
----
-
-### Compressed Files
-
-.pull-left[
-
-- Some files may be compressed/archived
-
-- Requires decompression/extraction to process the data
-
-- May receive errors if left compressed
-
-<br/><br/>
-
-- **File extensions:**
-  - .zip
-  - .gz
-  - .tar
-  - .tar.gz
-
-]
-
-.pull-right[
-
-![Basic structure of single-cell data](../../images/scrna-data-formats/unzip-tool.png)
-
-]
-
-???
-
-It is common for single-cell data to be stored in a compressed format in order to reduce filesizes and make transferring the data simpler. It is important to recognise when a file is compressed as using compressed data in downstream tools may cause errors to occur.
-
-There are two forms of compression that are common:
-
-- Individual file compression
-- Archive of multiple files
-
-Individual files are typically compressed with zip (.zip) or gzip (.gz) and should be unzipped prior to processing.
-
-When data is stored in multiple files/folders they may be archived into a single file (commonly called a tarball), this is done with the TAR tool (.tar). Like individual file compression, you will need to untar/extract the .tar file before further processing.
-
-Finally, it is also common for multiple files/folders to both be compressed and archived into a single file, this is indicated with the extension .tar.gz. In this case the data will need to be **both** unzipped and then extracted before the data is available in it's original form.
+---
+layout: tutorial_slides
+logo: GTN
+title: Single-cell Formats and Resources
+questions:
+- What does single-cell data look like?
+- What are the common single-cell formats?
+- Where can public single-cell data be found?
+objectives:
+- Learn the basic structure of single-cell data
+- Be able to recognise different types of single-cell metadata
+- Learn the common data formats used for storing single-cell data
+- Become familiar with some popular sources for public single-cell data
+subtopic: scintroduction
+priority: 4
+tags: 
+key_points:
+- Single-cell data consists of four main sections
+- Single-cell data can come in a variety of formats
+- Different data formats are required for different languages and packages
+- There are various public single-cell atlases that consist of useful data
+contributors:
+- hexhowells
+- nomadscientist
+- kkamieniecka
+requirements:
+- type: internal
+  topic_name: single-cell
+  tutorials:
+  - scrna-intro
+recordings:
+- youtube_id: _GsIlvWxdFw
+  length: 18M
+  galaxy_version: N/A
```

Comment:
@shiltemann and @hexylena: what are the guidelines for the Galaxy version for a slide deck recording submission?

Reply:
just remove the entire line if it is not relevant for the type of video

Reply:
Thanks!